addons/account/i18n/vi.po

```diff
@@ -10023,6 +10023,8 @@ msgid ""
 "You cannot change the value of the reconciliation on this account as it "
 "already has some moves"
 msgstr ""
+"Bạn không thể thay đổi giá trị của trường Cho phép Đối soát của tài khoản "
```

Comment:
please don't change that in github, instead make the modification on Transifex, cf [translation guidelines](https://github.com/odoo/odoo/wiki/Translations)